ProcessLib/LiquidFlow/LiquidFlowProcess.h

```diff
@@ -88,7 +101,7 @@ class LiquidFlowProcess final : public Process
 
     const int _gravitational_axis_id;
     const double _gravitational_acceleration;
-    LiquidFlowMaterialProperties _material_properties;
+    const std::unique_ptr<LiquidFlowMaterialProperties> _material_properties;
```

Comment:
As explained above,  getFluidProperties() is now changed to return a pointer. Therefore, I prefer to keep this declaration as a concrete instance. While the local assembler of LiquidFlow takes  a reference to this unique_ptr member.